.jenkins/pytorch/build.sh

```diff
@@ -71,7 +71,12 @@ if [[ "$BUILD_ENVIRONMENT" == *trusty-py3.6-gcc5.4* ]]; then
   export DEBUG=1
 fi
 
-WERROR=1 python setup.py install
+if [[ "$BUILD_ENVIRONMENT" == *ppc64le* ]]; then
+  export TORCH_CUDA_ARCH_LIST=6.0
```

Comment:
Put this up with the other TORCH_CUDA_ARCH_LIST. And I guess you can move `WERROR=1` out into an actual export, so you can write it conditionally without copying `python setup.py install`.